sdk/aqueduct/backend/api_client.py

```diff
@@ -272,6 +275,86 @@ def connect_integration(
         resp = requests.post(url, url, headers=headers)
         self.raise_errors(resp)
 
+    def get_dag_engine_status(
```

Comment:
Can we call this `get_dynamic_engine_status_by_dag()`? And the one below just `get_dynamic_engine_status()`?  I think `dag_engine` is a little confusing because isn't not just looking for the K8s engine set on the dag object, but also on each operator within the dag.